execute/table_test.go

```diff
@@ -386,6 +388,206 @@ func TestCopyTable(t *testing.T) {
 	}
 }
 
+func TestColListTableBuilder_AppendValues(t *testing.T) {
+	for _, tt := range []struct {
+		name   string
+		typ    flux.ColType
+		values array.Interface
+		want   *executetest.Table
+	}{
+		{
+			name: "Ints",
+			typ:  flux.TInt,
+			values: func() array.Interface {
+				b := array.NewInt64Builder(memory.DefaultAllocator)
+				b.Append(2)
+				b.Append(3)
+				b.AppendNull()
+				b.Append(8)
+				b.AppendNull()
+				b.Append(6)
+				return b.NewArray()
+			}(),
+			want: &executetest.Table{
+				ColMeta: []flux.ColMeta{{
+					Label: "_value",
+					Type:  flux.TInt,
+				}},
+				Data: [][]interface{}{
+					{int64(2)}, {int64(3)}, {nil}, {int64(8)}, {nil}, {int64(6)},
+				},
+			},
```

Comment:
I'm assuming these arrays have to be homogenous? 

If so, thanks for adding this test coverage. If not, maybe add an array with mixed types?

Reply:
Yes. They all have to be the same type.